File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


def main():
    height = get_height()
    for i in range(height):
        for j in range(height-i-1):
            print(" ", end="")
        for k in range(i+1):
            print("#", end="")
        print()


def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            break
    return n


main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)

{
    int n;
    do
    {
      n = get_int("Height:\n");
    }
  while (n < 1 || n > 8);

// For each row
  for (int i=0; i <n; i++)
  {

    // Print spaces for each column
      for (int j=i; j < n-1; j++)
      {
          printf(" ");
      }
    // Print Hashes for each column
      for (int j=-1; j < i; j++)
      {
          printf("#");
      }
      printf("\n");
  }
}